wmn-data.json

```diff
@@ -5467,6 +5563,18 @@
         "cat" : "XXXPORNXXX",
         "valid" : true
        },
+       {
+        "name" : "Subscribestar",
+        "uri_check" : "https://subscribestar.adult/{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "SUBSCRIPTION TIERS",

```

Comment:
I do not see this `e_string` in the results. please check it and maybe consider using a different one like `profile_main_info-name